Financial-Times/x-dash
# GitHub bot for aiding component PR workflow

to think about:

- [ ] how should this be deployed
- [ ] there's no way to set up project automation using the API. i _could_, but i won't, replicate the workflow using webhooks. lot of work and it doesn't do things like the progress bars which would be useful

if you want to test it, it's currently set up on https://github.com/financial-times-sandbox/Western-Storm (running on my local machine). suggested steps:

1. Create a PR and add the `Component` label to it
2. Observe the bot's comment
3. Create another PR from the first PR, and add the component's label
4. Play around with the projects
5. Try and break any assumptions the above steps have made

It might be possible to make the workflow simpler, e.g. by getting rid of the component-sub-PR label and managing all PRs that are based on another component PR. Thoughts?
